service/src/test/java/greencity/rating/RatingCalculationTest.java

```diff
@@ -60,7 +62,6 @@ void ratingCalculation() {
             .pointsChanged(rating.getRatingPoints())
             .build();
         when(modelMapper.map(userVO, User.class)).thenReturn(user);
-        doNothing().when(restClient).save(userVO, null);
```

Comment:
please remove `restClient`

Reply:
should we replace `restClient -> userService`

Reply:
yes, fixed